Model/Behavior/AuditableBehavior.php

```diff
@@ -60,6 +60,7 @@ public function setup(Model $Model, $settings = array()) {
 		}
 
 		if (!isset($this->settings[$Model->alias])) {
+            $this->settings[$Model->alias]['data_source'] = Configure::read('AuditLog.data_source');
```

Comment:
If we use the normal and established way of configuration for this plugin, we can use a fall back to ``default``.

I would introduce a property named ``$defaultDataSource`` set to ``default``.